platforms/tktrex/backend/lib/parser.ts

```diff
@@ -109,7 +109,7 @@ export const buildMetadata: BuildMetadataFn<
         metrics,
         stitch,
         author,
-        downloader,
+        ...downloader,
```

Comment:
are you sure this is right? before if was like this, and it was good:
![image](https://user-images.githubusercontent.com/89555/189966715-84432bd3-a7ee-4711-b5eb-d66f197701d3.png)

Reply:
Yep, I restored that behaviour, cause the output - before this change - was `downloader: { downloader: [] }`